frameworks/C++/lithium/lithium.cc

```diff
@@ -111,16 +113,31 @@ int main(int argc, char* argv[]) {
 #if TFB_MYSQL
     for (int i = 0; i < N; i++)
       c.update(numbers[i]);
+    raw_c("COMMIT");
 #elif TFB_PGSQL
-    std::ostringstream ss;
-    ss << "UPDATE World SET randomNumber=tmp.randomNumber FROM (VALUES ";
-    for (int i = 0; i < N; i++)
-      ss << "(" << numbers[i].id << ", " << numbers[i].randomNumber << ") "<< (i == N-1 ? "": ",");
-    ss << ") AS tmp(id, randomNumber) WHERE tmp.id = World.id";
-    raw_c(ss.str());
+
+    if (N == 20)
```

Comment:
@matt-42 Can you remove this and pick one implementation? We verify the results at N == 20 meaning your implementation in the else block would not be verified.